packages/react/carbon.yml

```diff
@@ -33,6 +33,11 @@ assets:
         name: Storybook
         action: link
         url: https://react.carbondesignsystem.com/?path=/docs/components-accordion
+    docs:
+      usagePath: https://github.com/carbon-design-system/carbon-website/blob/main/src/pages/components/accordion/usage.mdx
+      stylePath: https://github.com/carbon-design-system/carbon-website/blob/main/src/pages/components/accordion/style.mdx
+      accessibilityPath: https://github.com/carbon-design-system/carbon-website/blob/main/src/pages/components/accordion/accessibility.mdx
+      overviewPath: './src/components/Accordion/docs/overview.mdx'
```

Comment:
Just a quick comment for the reviewers - once https://github.com/carbon-design-system/carbon-platform/issues/1252 is closed, we'll only need `overviewPath` here, since `usagePath`, `stylePath`, and `accessibilityPath` get inherited from the `Carbon Core` (`@carbon/styles`) library. Adding all the other non-`overviewPath` paths is just temporary.